dist/api_data.py

```diff
@@ -1740,6 +1736,31 @@ def __ge__(self, other):
         dropped if open in a cursor. While a hot backup is in progress, checkpoints created
         prior to the start of the backup cannot be dropped''',
         type='list'),
+    Config('flush_tier', '', r'''
+        configure flushing objects to tiered storage after checkpoint''',
+        type='category', subconfig= [
+            Config('enabled', 'false', r'''
+                if true, perform one iteration of object switching and flushing objects to
+                tiered storage''',
+                type='boolean'),
+            Config('force', 'false', r'''
+                if false (the default), flush_tier may be skipped if the underlying object
```

Comment:
This feels ambiguous.  Does it mean `"flush_tier on any individual object may be skipped if the object...."` or `"flush_tier may be skipped if the underlying objects have not been modified...."`

Reply:
Fixed